hadoop-hdds/common/src/main/conf/hadoop-env.sh

```diff
@@ -400,7 +400,16 @@ export HADOOP_OS_TYPE=${HADOOP_OS_TYPE:-$(uname -s)}
 # These options will be appended to the options specified as HADOOP_OPTS
 # and therefore may override any similar flags set in HADOOP_OPTS
 #
-# export HDFS_OM_OPTS=""
+# export OZONE_MANAGER_OPTS=""
```

Comment:
I think `OZONE_OM_OPTS` is aligned with the naming conventions. Especially as it's already working thanks to the 

```
hadoop_subcommand_opts "${HADOOP_SHELL_EXECNAME}" "${HADOOP_SUBCMD}"
```

line.

Reply:
Renamed to `OZONE_OM_OPTS`